vars.yml

```diff
@@ -37,50 +33,15 @@ ibmcloud_image_ids:
     "rhel_8.3": "r042-71b7a0b1-5b3c-4024-b815-2b2d79687792"
     "rhel_8.4": "r042-06ad9b4d-02a7-44bb-989e-18f1882646ac"
 
-ibmcloud_vsi_profile: 'bx2-4x16'
-ibmcloud_vsi_count: 2
-install_tower: True
 # Find the right RHEL 8.3 image for the region
 # Recorded from for region in $IC_REGIONS; do ibmcloud target -r $region && ibmcloud is image ibm-redhat-8-3-minimal-amd64-3;
-ibmcloud_vsi_image_id: ibmcloud_image_ids[ibmcloud_vpc_region][ibmcloud_image]
-
-# Ansible Tower and Tower database must be in the same
-ibmcloud_vsi_db_profile: 'bx2-4x16'
-
-
-# Isolated Nodes must be in the different region compared to Controller
-ibmcloud_vpc_iso_name_prefix: 'ansible-tower-iso-test'
-ibmcloud_vpc_iso_region: 'eu-gb'    # London
-ibmcloud_vsi_iso_image_id: ibmcloud_image_ids[ibmcloud_vpc_iso_region][ibmcloud_image]
-ibmcloud_vsi_iso_profile: 'bx2-4x16'
-ibmcloud_vsi_iso_instance_count: 1
-install_iso: False
-
-
-# Execution Node Variables, in Tower 4.1, execution_node is a replacement of isolated nodes
-
-ibmcloud_vpc_execution_name_prefix: 'ansible-tower-exec-test'
-ibmcloud_vpc_execution_region: 'eu-gb'    # London
-ibmcloud_vsi_execution_image_id: ibmcloud_image_ids[ibmcloud_vsi_execution_region][ibmcloud_image]
-ibmcloud_vsi_execution_profile: 'bx2-4x16'
-ibmcloud_vsi_execution_instance_count: 1
-install_execution_node: False
-
-#_NODE_VALID_TYPES = {
-#    _CONTROL_PLANE: {
-#        "types": frozenset(("control", "hybrid")),
-#        "default_type": "hybrid",
-#    },
-#    _EXECUTION_PLANE: {
-#        "types": frozenset(("execution", "hop")),
-#        "default_type": "execution",
-
-#receptor_workable_types:
-#  - control
-#  - execution
-#  - hybrid
+ibmcloud_vsi_image_id: '{{ ibmcloud_image_ids[ibmcloud_vpc_region][ibmcloud_image] }}'
 

```

Comment:
Which file are these going into ? Are you moving them into a separate file ?

Reply:
```    - name: Fetch the tower variables from var file
      include_vars:
        file: install_tower_vars.yml
      when: install_tower|default(false)
      tags: debug```

I am not able to find the  `install_tower_vars.yml`.

Reply:
fixed